storage.tf

```diff
@@ -31,6 +33,35 @@ resource "aws_db_instance" "main" {
   }
 }
 
+resource "aws_db_instance" "from-snapshot" {
+  count = length(var.postgresql_source_snapshot_identifier) == 0 ? 0 : 1
+
+  apply_immediately      = var.postgresql_apply_immediately
+  identifier             = var.postgresql_name
+  storage_type           = var.postgresql_storage_type
+  instance_class         = var.postgresql_instance_class
+  username               = var.postgresql_username
```

Comment:
@WinnyTroy @DavisRayM could we confirm that we need the username and password variables for RDS instances that are being created from snapshots.

Reply:
The username and password are not required for databases built from snapshots. They utilize the originals username and password.

CC: @WinnyTroy

Reply:
Cool, I'll remove those values from this resource